src/supports_colors.rs

```diff
@@ -1,5 +1,6 @@
 use core::fmt;
 
+#[cfg(feature = "supports-colors")]
```

Comment:
Hmm I don't think you need this given that the module is gated under this feature? https://github.com/jam1garner/owo-colors/blob/37397e6d9a10de5009f66b5f0c89a70034a91841/src/lib.rs#L446-L447

Reply:
Yep! It's just a duplicate so that rustdoc will annotate it properly in documentation, as it currently won't auto-apply doc(cfg()) through modules.